src/main/java/se/sundsvall/casedata/api/model/conversation/MessageRequest.java

```diff
@@ -0,0 +1,26 @@
+package se.sundsvall.casedata.api.model.conversation;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(setterPrefix = "with")
+@Schema(description = "MessageRequest model")
+public class MessageRequest {
```

Comment:
Why not use Message (annotated with read only on all attributes besides inReplyToMessageId and content) instead of creating MessageRequest?